.github/workflows/vote-monitor.yml

```diff
@@ -55,6 +55,19 @@ jobs:
                 const projectName = sandboxMatch[1].trim();
                 console.log(`📝 Processing issue #${issue.number}: "${projectName}"`);
                 
+                const hasUnsignedCLA = (issue.labels || []).some(l => (typeof l === 'string' ? l : l.name) === 'contribution-agreement/unsigned');
```

Comment:
@riaankleinhans if this variable can be updated from `hasUnsignedCLA` to `hasUnsignedCA` this will be good to go.